Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp

```diff
@@ -299,6 +298,34 @@ namespace EditorPythonBindings
                     return EditorPythonBindings::Call::ClassMethod(method, m_wrappedObject, pythonArgs);
                 }
             }
+            else if (behaviorClass->m_unwrapper != nullptr)
```

Comment:
Reminder the PR with this change is available here: https://github.com/o3de/o3de/pull/14881/files#diff-dd5a173d37d1ab8bb213f818e28015645717496e629d83202c61b921b8fee399